portal/src/main/webapp/WEB-INF/jsp/cross_cancer_results.jsp

```diff
@@ -80,12 +80,15 @@
 
 <jsp:include page="global/header.jsp" flush="true"/>
 
+<script type="text/javascript" src="js/src/modifyQuery.js?<%=GlobalProperties.getAppVersion()%>"></script>
 
 <script>
 window.appVersion = '<%=GlobalProperties.getAppVersion()%>';
     
 window.historyType = 'memory';
 
+window.maxTreeDepth = '<%=GlobalProperties.getMaxTreeDepth()%>';
```

Comment:
Let's move this to `template.tag` if possible. Nice to keep all properties in one spot. There are a bunch of other properties for the query component such as `priorityStudies`